samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -584,15 +584,15 @@ Building and running
 Building with nRF7002 EK Wi-Fi scanning support (for nRF9160 DK)
 ================================================================
 
-To build the sample with nRF7002 EK Wi-Fi scanning support, use the ``-DSHIELD=nrf7002ek_nrf7002``, ``-DDTC_OVERLAY_FILE=nrf9160dk_with_nrf7002ek.overlay`` and  ``-DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf`` options.
+To build the sample with nRF7002 EK Wi-Fi scanning support, use the ``-DSHIELD=nrf7002ek``, ``-DDTC_OVERLAY_FILE=nrf9160dk_with_nrf7002ek.overlay`` and  ``-DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf`` options.
 
 This enables the Wi-Fi location tracking method automatically.
 
 For example:
 
 .. code-block:: console
 
-   west build -p -b nrf9160dk_nrf9160_ns -- -DSHIELD=nrf7002ek_nrf7002 -DDTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay" -DOVERLAY_CONFIG="overlay-nrf7002ek-wifi-scan-only.conf"
+   west build -p -b nrf9160dk_nrf9160_ns -- -DSHIELD=nrf7002ek -DDTC_OVERLAY_FILE="nrf9160dk_with_nrf7002ek.overlay" -DOVERLAY_CONFIG="overlay-nrf7002ek-wifi-scan-only.conf"
```

Comment:
I see that you are undoing the naming change from `nrf7002ek` to `nrf7002ek_nrf7002` that was introduced in https://github.com/nrfconnect/sdk-nrf/pull/10964 / https://github.com/nrfconnect/sdk-nrf/pull/11189

That is definitely intentional, right?

Reply:
Yes.